packages/quick-edit/functions/_middleware.ts

```diff
@@ -5,6 +5,8 @@ export const onRequest = async ({
 }: Parameters<PagesFunction>[0]) => {
 	const url = new URL(request.url);
 
+	const isLocalDev = url.hostname === "localhost";
```

Comment:
Is this a reliable enough way to compute local dev mode?
What if it was 127.0.0.1 or similar?
Do we care? Perhaps not.

Reply:
We don't really care—this is just for local development and it'll probably always be `localhost`